### ISSUE TITLE
Docker image build fails for python > 3.10

### ISSUE THREAD

**Issue by karyeet** [NateM135/UCR-Discord-Email-Verification#1]
```
Docker image build fails for python > 3.10
[+] Building 13.7s (8/10)                                                                                                          docker:default
 => [ucr-email-bot internal] load build definition from Dockerfile                                                                           0.0s
 => => transferring dockerfile: 266B                                                                                                         0.0s
 => [ucr-email-bot internal] load metadata for docker.io/library/python:latest                                                               0.6s
 => [ucr-email-bot internal] load .dockerignore                                                                                              0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [ucr-email-bot internal] load build context                                                                                              0.0s
 => => transferring context: 5.02kB                                                                                                          0.0s
 => [ucr-email-bot 1/6] FROM docker.io/library/python:latest@sha256:3966b81808d864099f802080d897cef36c01550472ab3955fdd716d1c665acd6         0.0s
 => CACHED [ucr-email-bot 2/6] COPY . ./usr/app/src                                                                                          0.0s
 => CACHED [ucr-email-bot 3/6] WORKDIR /usr/app/src                                                                                          0.0s
 => ERROR [ucr-email-bot 4/6] RUN pip install -r requirements.txt                                                                           13.1s
------
 > [ucr-email-bot 4/6] RUN pip install -r requirements.txt:
2.283 Collecting aiohttp==3.7.4.post0 (from -r requirements.txt (line 1))
2.335   Downloading aiohttp-3.7.4.post0.tar.gz (1.1 MB)
2.383      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 25.2 MB/s eta 0:00:00
2.472   Installing build dependencies: started
4.076   Installing build dependencies: finished with status 'done'
4.077   Getting requirements to build wheel: started
4.366   Getting requirements to build wheel: finished with status 'done'
4.368   Installing backend dependencies: started
6.063   Installing backend dependencies: finished with status 'done'
6.064   Preparing metadata (pyproject.toml): started
6.319   Preparing metadata (pyproject.toml): finished with status 'done'
6.341 Collecting async-timeout==3.0.1 (from -r requirements.txt (line 2))
6.345   Downloading async_timeout-3.0.1-py3-none-any.whl.metadata (4.0 kB)
6.367 Collecting asyncio==3.4.3 (from -r requirements.txt (line 3))
6.371   Downloading asyncio-3.4.3-py3-none-any.whl.metadata (1.7 kB)
6.398 Collecting attrs==21.4.0 (from -r requirements.txt (line 4))
6.401   Downloading attrs-21.4.0-py2.py3-none-any.whl.metadata (9.8 kB)
6.432 Collecting certifi==2021.10.8 (from -r requirements.txt (line 5))
6.435   Downloading certifi-2021.10.8-py2.py3-none-any.whl.metadata (3.0 kB)
6.457 Collecting chardet==4.0.0 (from -r requirements.txt (line 6))
6.459   Downloading chardet-4.0.0-py2.py3-none-any.whl.metadata (3.5 kB)
6.539 Collecting charset-normalizer==2.0.10 (from -r requirements.txt (line 7))
6.543   Downloading charset_normalizer-2.0.10-py3-none-any.whl.metadata (11 kB)
6.571 Collecting dnspython==2.1.0 (from -r requirements.txt (line 8))
6.575   Downloading dnspython-2.1.0-py3-none-any.whl.metadata (1.9 kB)
6.597 Collecting ffmpeg-python==0.2.0 (from -r requirements.txt (line 9))
6.601   Downloading ffmpeg_python-0.2.0-py3-none-any.whl.metadata (1.7 kB)
6.627 Collecting future==0.18.2 (from -r requirements.txt (line 10))
6.632   Downloading future-0.18.2.tar.gz (829 kB)
6.649      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 829.2/829.2 kB 55.2 MB/s eta 0:00:00
6.737   Preparing metadata (setup.py): started
7.109   Preparing metadata (setup.py): finished with status 'done'
7.132 Collecting idna==3.3 (from -r requirements.txt (line 11))
7.135   Downloading idna-3.3-py3-none-any.whl.metadata (9.8 kB)
7.163 Collecting imageio-ffmpeg==0.4.5 (from -r requirements.txt (line 12))
7.167   Downloading imageio_ffmpeg-0.4.5-py3-none-manylinux2014_aarch64.whl.metadata (1.6 kB)
7.188 Collecting mailjet-rest==1.3.4 (from -r requirements.txt (line 13))
7.192   Downloading mailjet_rest-1.3.4-py3-none-any.whl.metadata (9.6 kB)
7.368 Collecting multidict==5.2.0 (from -r requirements.txt (line 14))
7.373   Downloading multidict-5.2.0.tar.gz (49 kB)
7.381      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.0/49.0 kB 6.2 MB/s eta 0:00:00
7.400   Installing build dependencies: started
8.903   Installing build dependencies: finished with status 'done'
8.907   Getting requirements to build wheel: started
9.132   Getting requirements to build wheel: finished with status 'done'
9.135   Preparing metadata (pyproject.toml): started
9.367   Preparing metadata (pyproject.toml): finished with status 'done'
9.547 Collecting numpy==1.22.0 (from -r requirements.txt (line 15))
9.551   Downloading numpy-1.22.0.zip (11.3 MB)
9.683      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.3/11.3 MB 84.9 MB/s eta 0:00:00
10.17   Installing build dependencies: started
12.47   Installing build dependencies: finished with status 'done'
12.48   Getting requirements to build wheel: started
12.61   Getting requirements to build wheel: finished with status 'error'
12.61   error: subprocess-exited-with-error
12.61
12.61   × Getting requirements to build wheel did not run successfully.
12.61   │ exit code: 1
12.61   ╰─> [33 lines of output]
12.61       Traceback (most recent call last):
12.61         File "/usr/local/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
12.61           main()
12.61         File "/usr/local/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
12.61           json_out['return_val'] = hook(**hook_input['kwargs'])
12.61                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
12.61         File "/usr/local/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 112, in get_requires_for_build_wheel
12.61           backend = _build_backend()
12.61                     ^^^^^^^^^^^^^^^^
12.61         File "/usr/local/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 77, in _build_backend
12.61           obj = import_module(mod_path)
12.61                 ^^^^^^^^^^^^^^^^^^^^^^^
12.61         File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
12.61           return _bootstrap._gcd_import(name[level:], package, level)
12.61                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
12.61         File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
12.61         File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
12.61         File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
12.61         File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
12.61         File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
12.61         File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
12.61         File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
12.61         File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
12.61         File "<frozen importlib._bootstrap_external>", line 995, in exec_module
12.61         File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
12.61         File "/tmp/pip-build-env-3ispvi8s/overlay/lib/python3.12/site-packages/setuptools/__init__.py", line 16, in <module>
12.61           import setuptools.version
12.61         File "/tmp/pip-build-env-3ispvi8s/overlay/lib/python3.12/site-packages/setuptools/version.py", line 1, in <module>
12.61           import pkg_resources
12.61         File "/tmp/pip-build-env-3ispvi8s/overlay/lib/python3.12/site-packages/pkg_resources/__init__.py", line 2172, in <module>
12.61           register_finder(pkgutil.ImpImporter, find_on_path)
12.61                           ^^^^^^^^^^^^^^^^^^^
12.61       AttributeError: module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter'?
12.61       [end of output]
12.61
12.61   note: This error originates from a subprocess, and is likely not a problem with pip.
12.61 error: subprocess-exited-with-error
12.61
12.61 × Getting requirements to build wheel did not run successfully.
12.61 │ exit code: 1
12.61 ╰─> See above for output.
12.61
12.61 note: This error originates from a subprocess, and is likely not a problem with pip.
------
failed to solve: process "/bin/sh -c pip install -r requirements.txt" did not complete successfully: exit code: 1

```

**Comment by karyeet** [comment]
Is this project dead?

**Comment by n4t3m** [comment]
Fixed in #2
